### ISSUE TITLE
Error: net::ERR_BLOCKED_BY_CLIENT at chrome-extension://

### ISSUE THREAD

**Issue by savi8sant8s** [mujo-code/puppeteer-headful#120]
Hello guys. My name is Sávio, I'm from Brazil and I'm using Plasmo for my graduation project. I'm using cucumber and puppeteer to perform the extension's e2e tests and locally I can make it work. But I would like to enable a Github action to run these tests at the time of pull requests. I tried to use this action but when entering the page this error is triggered: Error: net::ERR_BLOCKED_BY_CLIENT at chrome-extension://.
Thank you in advance for your attention and if you can help.

Repository: https://github.com/bingobongo-testing/extension.

Github Action Error: https://github.com/bingobongo-testing/extension/actions/runs/3315466559/jobs/5476094414

**Comment by jcblw** [comment]
Thanks for opening up an issue. Looking into the error, it looks like that error is thrown if the puppeteer has disabling extensions or you are opening up in a headless mode. I looked at the code, and it looks correct to me.

Can you try filtering out the disable extensions argument, and see if that works for you?

https://github.com/puppeteer/puppeteer/issues/1788#issuecomment-357150689

**Comment by savi8sant8s** [comment]
I went on the Discord of the Plasmo Framework community and also asked for help there.

I discovered now that I was passing a public key in the secret that was not being identified at the time of compiling the project with Plasmo. I will find out why.

Passing directly in package.json worked and thus a unique id for the Chrome extension is generated. Thanks a lot for the help. Have a nice day and much success.

It works!!
https://github.com/bingobongo-testing/extension/actions/runs/3315879090/jobs/5477018581
